float64 x
float64 y
float64 z

float64 yaw
float64 pitch
float64 roll

float64 pos_accuracy 
float64 angle_accuracy 
float64 angle_rate

int8 control_flag

int8 cv_control_flag

float64 cv_x
float64 cv_y
float64 cv_z
float64 cv_yaw

float64 cv_pos_accuracy 
float64 cv_angle_accuracy 
float64 cv_angle_rate

int8 cv_times